internal/trace/spancontext.go

```diff
@@ -0,0 +1,116 @@
+/*
+This SpanContext is implements
+[ddtrace.SpanContextW3C](https://github.com/DataDog/dd-trace-go/blob/49273dfcc871f3329614c40cd98116b6d44cdb12/ddtrace/ddtrace.go#L25)
+, implementation is mostly based on
+https://github.com/DataDog/dd-trace-go/blob/49273dfcc871f3329614c40cd98116b6d44cdb12/ddtrace/tracer/spancontext.go#L85
+*/
```

Comment:
Can we add why we needed to copy it over here?

Reply:
:+1: